hack/generate-metrics-doc.go

```diff
@@ -78,4 +94,6 @@ func main() {
 
 		f.Close()
 	}
+
+	runPromlint(family)
```

Comment:
Maybe comment this out for now with a TODO (and link a relevant issue) since this will cause the code generation job to fail

Reply:
fixed.